Makefile

```diff
@@ -9,15 +9,15 @@ composer:
 	@$(RUNNER) "composer $(filter-out $@,$(MAKECMDGOALS))"
 dependencies:
 	make -s composer update -- --prefer-dist
-test: dependencies
+test:
```

Comment:
Why not dependent on updating dependencies anymore?

Reply:
Installing dependencies before each command is good practice but not required. But unlike `go get`, `composer update` is very slow and I've to wait like 5-10 seconds before seeing any test output. Go frustrated and removed it.